lms/services/user.py

```diff
@@ -107,9 +107,9 @@ def get_users(  # noqa: PLR0913, PLR0917
         role_scope: RoleScope,
         role_type: RoleType,
         instructor_h_userid: str | None = None,
-        course_id: str | None = None,
```

Comment:
This was marked as str but is an int. mypy doesn't catch it because it doesn't know about marshmallow types.